docs/zkapps/advanced-snarkyjs/permissions.mdx

```diff
@@ -0,0 +1,294 @@
+
+`setDelegate`: The Permission corresponding to the ability to set the delegate field of the account.
+The delegate field is the address of another account, to whom this account is delegating its MINA to for staking.
+
+`setPermissions`: The Permission corresponding to the ability to change the permissions of the account.
+As the name might suggest, this type of permission describes how already set permissions can be changed.
+
+`setVerificationKey`: The Permission corresponding to the ability to change the verification key of the account.
+Every smart contract has a verification key stored on-chain. The verification key is used to verify off-chain proofs. This permission essentially describes if the verification key can be changed or not - we can also think of it as "upgradeability" of smart contracts.
+
+`setZkappUri`:  The Permission corresponding to the ability to change the zkapp uri field of the account.
+The `zkappUri` is a field that can be used to store some meta data about the smart contract (e.g. link to the source code). The permission determines how this field can be changed.
+
+`editSequenceState`: The Permission corresponding to the ability to change the sequence state of the associated account.
+Every smart contract can dispatch sequence events (or actions), which are committed to on-chain. This type of permission describes who can change that value.
+
+`setTokenSymbol`: The Permission corresponding to the ability to set the token symbol for this account.
+Similar to `setZkappUri` and `zkappUri`, this field stored the symbol of a token.
+
+`incrementNonce`: The Permission that determine whether or not the nonce should be incremented with an account update.
+This Permission determines who can increment the nonce on this account with a transaction.
+
+`setVotingFor`: The permissions corresponding to the ability to set the chain hash for this account. 
+The `votingFor` field is an on-chain mechanism to set the chain hash of the hard fork this account is voting for. 
+
+
+## Authorization
+
+Permissions just describe who has the ability to execute an action, but they don't authorize it.
+Currently, we have 5 types of authorizations: `none`, `impossible`, `signature`, `proof` and `proofOrSignature`.
+
+We already know that a transaction consists of multiple account updates (sort of like instructions to the network) - and each account update has to be authorized in one way or another.
+When you inspect an account update (either directly in SnarkyJS or using an explorer), you will notice a field `authorization` on the account update.
+If this field has a proof attached, it means it's authorized by a proof, which will be checked against the verification key of the account. If it's a signature, that means the account update was authorized by a signature.
+
+Here is an example of how this may look like: 
+
+```json
+{
```

Comment:
I think its important to briefly mention the structure (where does the authorization (sig, proof) come from? how is this enforced?)